package-readmes/animationHelpers.md

```diff
@@ -2,425 +2,416 @@
+
+    ```js
+    float($w('VectorImage'));
+    ```
 
-    +  Repeater 3: The `repeaterBtnOnly` property object defines the repeater and the button item you want to animate.  
 
-        ```js
-        const repeaterBtnOnly = {
-          self: $w('#repeaterBtnOnly'),
-          itemBtn: $w('#itemBtnOnly')
-        };
-        ```
+##### Float an element on hover  
 
-5. Build a property object for the `initRepeaterAnimation()` function to enable animation of each of the 3 repeaters for 0.3 seconds.
+1. Add a text element to your site. 
+2. Import the `float` function from the package to your page. 
 
     ```js
-    const repeatersAnimationProps = {
-      repeaters: [repeaterImageButton, repeaterImageOnly, repeaterBtnOnly],
-      options: {
-        imageScale: 1.1,
-        imageOpacity: 0.7,
-        duration: 300
-      }
-    };
+    import { float } from '@velo/wix-animation-helpers';
     ```
 
-6. Call the `initRepeaterAnimation()` function with the property object you created.
+3. Run your code in the `$w.onReady()` function.
+4. To float the text element on hover, we first need to stop the animation function from playing by default. To do this, call the `float()` function with the text element and change the default value of the `pause` parameter from `false` to `true`. We will also change the text element to float for a duration of 3 seconds.  
 
-    ```
-    initRepeaterAnimation(repeatersAnimationProps);
-    ```
+      ```js
+     const floatingText = $w('#floatingText');
+     const textFloatTimeLine = float(floatingText, { duration: 3000, direction: 'horizontal', pause: true });
+      ```
 
 
-7. For each item in the `repeaterImageButton` repeater, add the `onMouseIn()` event handler to the repeater’s container box. Then call the `runRepeaterAnimation` function with the items in the repeater you want to animate and the `play` parameter in the event handler to animate the repeater items on hover. 
 
-    ```js
-    $w('#repeaterImageButton').forEachItem(($item) => {
-      $item('#containerImageBtn').onMouseIn(() => {
-        runRepeaterAnimation({ itemImage: $item('#itemImage'), itemBtn: $item('#itemBtn'), action: 'play' })
-      });
+5. Add the `onMouseIn()` event handler to the text element. This function will run when you hover over the text element. Call the `textFloatTimeLine.play()` function in the event handler to play the animation on hover.  
+Add the `onMouseOut()` event handler to the text element. Call the `textFloatTimeLine.pause()` function in the event handler to pause the animation when you are no longer hovering over the text. 
+
+      ```js
+     floatingText.onMouseIn(() => {
```

Comment:
This block of code isn't showing properly. Play around with js tags and/or see where I placed them in the github package readme repo.